app/services/course_output_writer.rb

```diff
@@ -0,0 +1,15 @@
+# Write out output from Course APIs to a JSON file to be read in by the app
+class CourseOutputWriter
+  # Called on CourseOutputWriter.new
+  def initialize(file_term, courses)
+    @file_term = file_term
+    @courses = courses
+  end
+
+  # Providing parameter for connection enables testing with Faraday connection stub
+  def write
+    file_name = "course_#{@file_term}.json"
+    # Write the JSON to the course_work_content folder where the app will pick it up later
+    File.write("#{Rails.root}/lib/course_work_content/#{file_name}", @courses.to_json)
```

Comment:
```suggestion
    File.write(Rails.root.join('lib/course_work_content', file_name), @courses.to_json)
```

Reply:
When I tried this change, Rubocop had a suggestion so it now reads 
`Rails.root.join('lib/course_work_content', file_name).write(@courses.to_json)`

Reply:
Cool!